daemon/rpc_account.go

```diff
@@ -18,6 +18,18 @@ func (r *RPC) AccountInfo(ctx context.Context, _ *pb.Empty) (*pb.AccountResponse
 		return nil, internal.ErrNotLoggedIn
 	}
 
+	accountInfo := &pb.AccountResponse{}
+
+	vpnExpired, err := r.ac.IsVPNExpired()
+	if err != nil {
+		log.Println(internal.ErrorPrefix, "checking VPN expiration: ", err)
+		return &pb.AccountResponse{Type: internal.CodeTokenRenewError}, nil
```

Comment:
These internal error codes don't fit our current approach I think. Currently we are relying on dedicated protobuff responses defined for each RPC. I sometimes refactor it while I'm working on other stuff, but sometimes it results in a lot of extra work and bloated PR.

Reply:
The code changes aren't very extensive so I decided not to refactor this time.